interface/forms/newpatient/common.php

```diff
@@ -86,16 +86,35 @@ function refreshIssue(issue, title) {
   s.options[s.options.length] = new Option(title, issue, true, true);
  }
 
+ <?php
```

Comment:
In this simple form case, recommend replacing the current validation with your method (and then making the rule active by default in the Page Validation list). If do this, would it then be easy to change the form to support validateUsingPageRules mechanism?